src/Publisher.php

```diff
@@ -56,6 +56,11 @@ abstract class Publisher implements StaticPublisher
      */
     private static $add_timestamp = false;
 
+    /**
+     * Cache redirector pages which will cache as the page they're redirected to
+     */
+    private static $cache_redirector_pages = true;
```

Comment:
I'm leaving this a true by default for backwards compatibility, I'd probably prefer it to be false by default though - though that should really require a new major version